README.md

```diff
@@ -281,12 +281,10 @@ To do this, first uninstall the script:
 sudo phala uninstall
 ```
 
-And delete the mining script repository by executing:
+Delete the mining script repository (if still on your machine) by executing, you may first need to find your home directory, where `solo-mining-scripts-main` should reside. To find this out, execute: 
```

Comment:
Now it's just a single `rm`. So we don't need to mention finding the directory I believe

Reply:
Definitely. Just fixed it here: https://github.com/Phala-Network/solo-mining-scripts/pull/40/commits/1f2588ba865f0be2af391c76c8a7d361dad256a2